### ISSUE TITLE
Limited horizontal extraocular movement

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7130]
**Preferred term label:**
Limited horizontal extraocular movement

**Synonyms**

**Definition (free text, please give PubMed ID)**

This is a dx data dictionary entry in the Kids First project, GMKF: Kids First Pediatric Research Program on Congenital Cranial Dysinnervation Disorders and Related Birth Defects.

https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/dataset.cgi?study_id=phs001247.v1.p1&phv=393326&phd=&pha=&pht=8299&phvf=&phdf=&phaf=&phtf=&dssp=1&consent=&temp=1

**Parent term (use hpo.jax.org/app)**

Limited extraocular movements
HP:0007941

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Syndromic cranial dysinnervation disorders

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
adding as HP:0025720
